docs/sources/reference/commandline/cli.md

```diff
@@ -1832,8 +1837,39 @@ An example of a file passed with `--env-file`
 
     $ sudo docker run --name console -t -i ubuntu bash
 
-This will create and run a new container with the container name being
-`console`.
+A label is a a `key=value` pair that applies metadata to a container. To label a container with two labels:
+
+    $ sudo docker run -l my-label --label com.example.foo=bar ubuntu bash
+
+The `my-label` key doesn't specify so the label defaults to an empty
+string(`""`). To add multiple labels, repeat the label flag (`-l` or
+`--label`).
+
+The `key=value` must be unique. If you specify the same key multiple times
```

Comment:
if I can specify a key multiple times, then the `key=value` doesn't need to be unique - please remove this sentence :)

ie, I can have `docker run -l sven=yes -l sven=no -l sven=yes ubuntu bash` and docker will not error out.